<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BacklogManagerClass</class>
 <widget class="QMainWindow" name="BacklogManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BacklogManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="ViewTasks">
       <attribute name="title">
        <string>View Tasks</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="opaqueResize">
           <bool>true</bool>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QLineEdit" name="SearchTaskField">
           <property name="minimumSize">
            <size>
             <width>650</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="SearchButton">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="ShowCompletedCheckbox">
           <property name="text">
            <string>Show Completed</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="TaskView">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_5">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPushButton" name="PreviousPage">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;&lt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="NextPage">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&gt;&gt;</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CreateTask">
       <attribute name="title">
        <string>Create Task</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="HeaderLayout">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="TaskName">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="TaskNameEdit">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QComboBox" name="TaskType">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>BLOCKER</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BUG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FEATURE</string>
            </property>
           </item>
          </widget>
          <widget class="QComboBox" name="TaskPriority">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>VERY LOW</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LOW</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MEDIUM</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HIGH</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>VERY HIGH</string>
            </property>
           </item>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="ResolverHorizontalLayout">
              <item>
               <widget class="QLabel" name="ResolverLabel">
                <property name="text">
                 <string>Resolver</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ResolverEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="ReviewerHorizontalLayout">
              <item>
               <widget class="QLabel" name="ReviewerLabel">
                <property name="text">
                 <string>Reviewer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="ReviewerEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="EffortLayout">
              <item>
               <widget class="QLabel" name="EffortLabel">
                <property name="text">
                 <string>Effort</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="EffortEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="DueDateLayout">
              <item>
               <widget class="QLabel" name="DueDateLabel">
                <property name="text">
                 <string>Due Date</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="DueDateEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="BodyLayout">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="TaskDescriptionLabel">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="TaskDescriptionArea"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPushButton" name="CreateTaskButton">
           <property name="minimumSize">
            <size>
             <width>750</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Create</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="StartTaskButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Start Task</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="MoveTask">
       <attribute name="title">
        <string>Move Task</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>true</bool>
          </property>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>From</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit"/>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>To</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BacklogManager.qrc"/>
 </resources>
 <connections/>
</ui>
